test/integration/master/api/finance/billing_jobs_controller_test.rb

```diff
@@ -28,8 +28,6 @@ class Master::Api::Finance::BillingJobsControllerTest < ActionDispatch::Integrat
   end
 
   test 'trigger billing to all buyers of a provider' do
-    SphinxIndexationWorker.stubs(:perform_later)
-
```

Comment:
Here and at the places where `perform_enqueued_jobs(except:` was used, now this is not needed anymore because sphinx workers are globally disabled (stubbed) and enabled only where needed.